Cimetidine-Model.json

```diff
@@ -24462,43 +24463,43 @@
             {
               "MoleculeName": "CYP2C8"
             },
-            {
-              "Name": "Glomerular Filtration-Optimized",
-              "SystemicProcessType": "GFR"
-            },
             {
               "MoleculeName": "CYP3A4"
             },
             {
               "MoleculeName": "AADAC"
             },
             {
-              "MoleculeName": "P-gp"
+              "MoleculeName": "UGT1A4"

```

Comment:
I think the order is different but otherwise the content is the same.

Reply:
True, it's only the order